lib/src/widget/built_in/built_in.dart

```diff
@@ -84,44 +83,32 @@ class BuiltInContent extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    Widget content = Text(
-      title,
-      style: style.titleTextStyle(context)?.copyWith(
-            color: foregroundColor,
-          ),
-    );
+    Widget? content = title ?? const SizedBox();
```

Comment:
Hello, @callmephil 

Looks like you forgot to wrap the title widget with the `DefaultTextStyle`. At the moment, title text has this style: `style.titleTextStyle(context)?.copyWith(color: foregroundColor)` 🙂

Reply:
Thanks @Maksimka101 for your review 

I didn't recognize this one 

Fixed